funcfile.go

```diff
@@ -8,7 +8,7 @@ import (
 	"path/filepath"
 	"strings"
 
-	yaml "gopkg.in/yaml.v2"
```

Comment:
Can you keep these explicit ones in here, I don't know what it is, but my tools (`goimports` maybe?) often rewrite these to the github URL.

Reply:
this particular import works fine without any alias, so, why it is needed?

Reply:
anyways, reverted.